browser/ui/BUILD.gn

```diff
@@ -4,94 +4,85 @@ import("//chrome/common/features.gni")
-    "webui/settings/brave_relaunch_handler_mac.mm",
-    "webui/settings/brave_relaunch_handler_mac.h",
-    "webui/settings/default_brave_shields_handler.cc",
-    "webui/settings/default_brave_shields_handler.h",
     "webui/sync/sync_ui.cc",
     "webui/sync/sync_ui.h",
-    "webui/settings/brave_default_extensions_handler.cc",
-    "webui/settings/brave_default_extensions_handler.h",
   ]
 
+  if (!is_android) {
+    sources += [
+      "brave_browser_command_controller.cc",
+      "brave_browser_command_controller.h",
+      "brave_browser_content_setting_bubble_model_delegate.cc",
+      "brave_browser_content_setting_bubble_model_delegate.h",
+      "brave_dark_mode_observer.cc",
+      "brave_dark_mode_observer.h",
+      "brave_layout_constants.cc",
+      "brave_layout_constants.h",
+      "brave_pages.cc",
+      "brave_pages.h",
+      "browser_commands.cc",
+      "browser_commands.h",
+      "content_settings/brave_autoplay_blocked_image_model.cc",
+      "content_settings/brave_autoplay_blocked_image_model.h",
+      "content_settings/brave_autoplay_content_setting_bubble_model.cc",
+      "content_settings/brave_autoplay_content_setting_bubble_model.h",
+      "content_settings/brave_content_setting_bubble_model.cc",
+      "content_settings/brave_content_setting_bubble_model.h",
+      "content_settings/brave_content_setting_image_models.cc",
+      "content_settings/brave_content_setting_image_models.h",
+      "omnibox/brave_omnibox_client.cc",
+      "omnibox/brave_omnibox_client.h",
+      "toolbar/brave_app_menu_model.cc",
+      "toolbar/brave_app_menu_model.h",
+      "webui/brave_md_settings_ui.cc",
+      "webui/brave_md_settings_ui.h",
+      "webui/settings/brave_privacy_handler.cc",
+      "webui/settings/brave_privacy_handler.h",
+      "webui/settings/brave_relaunch_handler_mac.mm",
+      "webui/settings/brave_relaunch_handler_mac.h",
+      "webui/settings/default_brave_shields_handler.cc",
+      "webui/settings/default_brave_shields_handler.h",
+    ]
+  }
+
+  if (toolkit_views) {
+    sources += [
+      "views/brave_layout_provider.cc",
+      "views/brave_layout_provider.h",
+      "views/download/brave_download_item_view.cc",
+      "views/download/brave_download_item_view.h",
+      "views/frame/brave_browser_view.cc",
+      "views/frame/brave_browser_view.h",
+      "views/importer/brave_import_lock_dialog_view.cc",
+      "views/importer/brave_import_lock_dialog_view.h",
+      "views/rounded_separator.cc",
+      "views/rounded_separator.h",
+      "views/toolbar/bookmark_button.cc",
+      "views/toolbar/bookmark_button.h",
+      "views/update_recommended_message_box_mac.mm",
+      "views/update_recommended_message_box_mac.h",
+    ]
+  }
+
+  if (is_win || is_mac || is_desktop_linux) {
+    sources += [
+      "views/profiles/brave_profile_chooser_view.cc",
+      "views/profiles/brave_profile_chooser_view.h",
+      "webui/brave_welcome_ui.cc",
+      "webui/brave_welcome_ui.h",
```

Comment:
I think we'll want to have welcome_ui available in Android, we can fix later though.